packages/neuron-ui/src/locales/en.json

```diff
@@ -272,7 +271,10 @@
       "rpc-url-should-have-protocol": "The RPC URL should start with http(s)://",
       "rpc-url-should-have-no-whitespaces": "The RPC URL should have no whitespaces",
       "is-required": "{{field}} is required",
-      "is-used": "{{field}} is used"
+      "is-used": "{{field}} is used",
+      "invalid-address": "{address}} is an invalid address",
+      "invalid-checksum": "{{address}} has an invalid checksum",
```

Comment:
We can use `invalid-address` translation for checksum too. People usually don't understand what checksum is.

Reply:
`invalid-address` and `invalid-checksum` has been removed in https://github.com/nervosnetwork/neuron/pull/883/commits/bba929219e50cf1eef300fde610d7291aedbaf88